### PR TITLE
Updated documentation

### PR DESCRIPTION
**Pull request by marcin-krolik**
Updated documentation

Summary of changes:
- updated example
- capitalized Snap

@intelsdi-x/plugin-maintainers

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -159,68 +162,23 @@ NAMESPACE                                                VERSIONS
...
-          }
-        }
-        ]
-      }
-    }
-}
 ```
 
-Create a task:
+Download an [example task file](https://github.com/intelsdi-x/snap-plugin-collector-influxdb/blob/master/examples/tasks/influxdb-file.json) and load it:
```
Comment (kindermoumoute) [inline README.md]: Change to a relative link `examples/tasks/influxdb-file.json`

File: README.md [hunk 2]
```diff
@@ -29,30 +29,32 @@ The plugin is used in the [snap framework] (http://github.com/intelsdi-x/snap).
...
 ### Operating systems
 
-All OSs currently supported by snap:
+All OSs currently supported by Snap:
 * Linux/amd64
 
 ### Installation
+#### Download the plugin binary:
+
+You can get the pre-built binaries for your OS and architecture from the plugin's [GitHub Releases](https://github.com/intelsdi-x/snap-plugin-collector-influxdb/releasess) page. Download the plugin from the latest release and load it into `snapd` (`/opt/snap/plugins` is the default location for Snap packages).
```
Comment (kindermoumoute) [inline README.md]: releases*

File: README.md [hunk 3]
```diff
@@ -29,30 +29,32 @@ The plugin is used in the [snap framework] (http://github.com/intelsdi-x/snap).
...
 $ make
 ```
-This builds the plugin in `/build/rootfs/`
+This builds the plugin in `./build/`
 
 ### Configuration and Usage
 
-* Set up the [snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
+* Set up the [Snap framework](https://github.com/intelsdi-x/snap/blob/master/README.md#getting-started)
 * Load the plugin and create a task, see example in [Examples](https://github.com/intelsdi-x/snap-plugin-collector-influxdb/blob/master/README.md#examples).
```
Comment (kindermoumoute) [inline README.md]: Change the url to `#examples`

File: README.md [hunk 4]
```diff
@@ -80,24 +82,25 @@ List of collected metrics is described in [METRICS.md](https://github.com/intels
 
 ### Examples
 
-Example of running snap influxdb collector and writing data to file.
-
-Run the snap daemon:
+Example of running Snap influxdb collector and writing data to file.
+Download an [example Snap global config](https://github.com/intelsdi-x/snap-plugin-collector-influxdb/blob/master/examples/configs/snap-config-sample.json) file.
```
Comment (kindermoumoute) [inline README.md]: Change the url to `examples/configs/snap-config-sample.json`

Comment (marcin-krolik) [inline README.md]: @kindermoumoute removing `blob/master` will lead to 404 Not Found.

File: README.md (hunk 1 above)
Comment (marcin-krolik) [inline README.md]: Same as above

File: README.md (hunk 3 above)
Comment (marcin-krolik) [inline README.md]: removing `blob/master` will lead to 404 Not Found. In `Snap` documentation this pattern for url is used as well

Comment (marcin-krolik) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (marcin-krolik) [inline README.md]: Done

**Comment by andrzej-k** [comment]
Please consider squashing into one commit, other than that LGTM.

**Pull request merged**